admin_guide/managing_networking.adoc

```diff
@@ -258,6 +258,14 @@ take up to one minute to adapt to these changes.
 # oc create -f <policy>.json
 ----
 
+[CAUTION]
+====
+Exposing services by creating xref:../dev_guide/routes.adoc#creating-routes[*routes*]
+will ignore `EgressNetworkPolicy`. We do egress network policy service endpoint filtering
```

Comment:
Egress network policy service endpoint filtering is done at node `kubeproxy`. When the router is involved, `kubeproxy` is bypassed and egress network policy enforcement is not applied.